ignored-tests.json

```diff
@@ -0,0 +1,179 @@
+  "Emulation Page.emulate should support clicking",
+  "Emulation Page.emulate should work",
+  "Emulation Page.emulateCPUThrottling should change the CPU throttling rate successfully",
+  "Emulation Page.emulateMediaFeatures should work",
+  "Emulation Page.emulateMediaType should work",
+  "Emulation Page.emulateNetworkConditions should change navigator.connection.effectiveType",
+  "Emulation Page.emulateTimezone should throw for invalid timezone IDs",
+  "Emulation Page.emulateTimezone should work",
+  "Emulation Page.emulateVisionDeficiency should work",
+  "Emulation Page.viewport should be detectable by Modernizr",
+  "Emulation Page.viewport should detect touch when applying viewport with touches",
+  "Emulation Page.viewport should support landscape emulation",
+  "Emulation Page.viewport should support touch emulation",
+  "Evaluation specs Page.evaluate should fail for circular object",
+  "Fixtures dumpio option should work with pipe option",
+  "ignoreHTTPSErrors Response.securityDetails Network redirects should report SecurityDetails",
+  "ignoreHTTPSErrors Response.securityDetails should be |null| for non-secure requests",
+  "ignoreHTTPSErrors Response.securityDetails should work",
+  "input tests FileChooser.accept should accept single file",
+  "input tests FileChooser.accept should be able to read selected file",
+  "input tests FileChooser.accept should be able to reset selected files with empty file list",
+  "input tests FileChooser.accept should error on read of non-existent files",
+  "input tests FileChooser.accept should fail when accepting file chooser twice",
+  "input tests FileChooser.accept should not accept multiple files for single-file input",
+  "input tests FileChooser.accept should succeed even for non-existent files",
+  "input tests FileChooser.cancel should cancel dialog",
+  "input tests FileChooser.cancel should fail when canceling file chooser twice",
+  "input tests FileChooser.isMultiple should work for \"multiple\"",
+  "input tests FileChooser.isMultiple should work for \"webkitdirectory\"",
+  "input tests FileChooser.isMultiple should work for single file pick",
+  "input tests Page.waitForFileChooser should prioritize exact timeout over default timeout",
+  "input tests Page.waitForFileChooser should respect default timeout when there is no custom timeout",
+  "input tests Page.waitForFileChooser should respect timeout",
+  "input tests Page.waitForFileChooser should return the same file chooser when there are many watchdogs simultaneously",
+  "input tests Page.waitForFileChooser should work when file input is attached to DOM",
+  "input tests Page.waitForFileChooser should work when file input is not attached to DOM",
+  "input tests Page.waitForFileChooser should work with no timeout",
+  "JSHandle Page.evaluateHandle should return the RemoteObject",
+  "Keyboard should send a character with sendCharacter in iframe",
```

Comment:
Isn't that one related to `frame.name` as well? I think we need that.

Reply:
Updating!